Jenkinsfile

```diff
@@ -174,6 +183,28 @@ pipeline {
             echo 'Skipping Kafka Streams archetype test for Java 21'
           }
         }
+
+        stage('PowerPC') {
+          agent { label 'ppc64le' }
+          tools {
+            jdk 'jdk_21_latest'
+          }
+          options {
+            timeout(time: 8, unit: 'HOURS')
+            timestamps()
+          }
+          environment {
+            SCALA_VERSION=2.13
+          }
+          when {
```

Comment:
Can you clarify why we are not using the `cron` statement? For example https://github.com/apache/zookeeper/blob/master/Jenkinsfile#L31-L33

Reply:
Not sure how to use `cron` for a specific stage. Maybe can you help out on this one?

Reply:
I was asking as I don't know either. But it does not seem to work: https://ci-builds.apache.org/blue/organizations/jenkins/Kafka%2Fkafka-pr/detail/PR-14717/7/pipeline

Reply:
@mimaison I have reverted the change, seem the previous logic one which I implemented is the only way to do it from my pov.